database/scrapers/contributors.ts

```diff
@@ -142,26 +140,12 @@ export default class ContributorsScraper extends BaseScraperModule {
       await Promise.all(
         Array.from(imageIds).map((id) =>
           this.semaphore.runTask(async () => {
-            const details = (await this.fetchJSON(
-              `https://admin.topwr.solvro.pl/files/${id}?fields=filename_disk`,
-              `details for file ${id}`,
-            )) as { data: { filename_disk: string } };
-            const extension = details.data.filename_disk
-              .split(".")
-              .pop()
-              ?.toLowerCase();
-            const fileResponse = await this.fetchAndCheckStatus(
-              `https://admin.topwr.solvro.pl/assets/${id}`,
-              `file ${id}`,
-            );
-            if (fileResponse.body === null) {
-              throw new Error(`Response for file ${id} has no body, wtf`);
-            }
-            const file = await FilesService.uploadStream(
-              Readable.fromWeb(fileResponse.body),
-              extension,
-            ).addErrorContext(() => `Failed to upload file ${id} to storage`);
-            return [id, file.id] as [string, string];
+            return [
+              id,
+              await this.directusUploadFieldAndGetKey(id).addErrorContext(
+                "Images from Contributors imageIds",
+              ),
```

Comment:
Any string passed to `.addErrorContext` becomes the new top-level error message, so make sure it makes sense as an error message.